compile_tools.sh

```diff
@@ -155,6 +155,17 @@ build_openh264() {
   cp testbin/layer2.cfg $TOOLDIR/layer2.cfg
 }
 
+build_libavc() {
+  # Build the Android M libavc
+  if [ ! -d libavc ]; then
+    git clone https://android.googlesource.com/platform/external/libavc
+  fi
+  # TODO(hta): Pick a version of libavc to compile. This compiles tip-of-tree.
+  (cd libavc; make -f ../../src/Makefile.libavcenclib)
```

Comment:
All the build_ functions start in third_party, which only contains stuff snarfed from other directories. I was told by the lawyers to put stuff I added to make what's in third_party work for me somewhere else (src/ in this case), so that's where the makefiles live.

I really wanted to do "make -f ../src/Makefile" (one makefile only), and put the results in some kind of "out/" directory, but the peculiarities of the Android makefiles (which are included by the makefiles I added) made this complex enough that I stopped trying.